spynnaker/pyNN/models/spike_source/spike_source_array_partitioned_vertex.py

```diff
@@ -1,15 +1,52 @@
-from spynnaker.pyNN.models.abstract_models.buffer_models\
-    .sends_buffers_from_host_partitioned_vertex_pre_buffered_impl\
-    import SendsBuffersFromHostPartitionedVertexPreBufferedImpl
+"""
+SpikeSourceArrayPartitionedVertex
+"""
+
+# pacman imports
+from pacman.model.data_request_interfaces.\
+    abstract_requires_routing_info_partitioned_vertex import \
+    RequiresRoutingInfoPartitionedVertex
 from pacman.model.partitioned_graph.partitioned_vertex import PartitionedVertex
 
+# spinn front end common imports
+from spinn_front_end_common.interface.buffer_management.buffer_models.\
+    sends_buffers_from_host_partitioned_vertex_pre_buffered_impl \
+    import SendsBuffersFromHostPartitionedVertexPreBufferedImpl
+
 
 class SpikeSourceArrayPartitionedVertex(
-        PartitionedVertex,
+        PartitionedVertex, RequiresRoutingInfoPartitionedVertex,
```

Comment:
RequiresRoutingInfoPartitionedVertex doesn't exist in PACMAN, and there is no branch.  Should this actually be in sPyNNaker, and is then just missing?